LEAF_Request_Portal/sources/Form.php

```diff
@@ -1403,8 +1403,9 @@ public function doSubmit(int $recordID): array
                         LEFT JOIN workflow_steps USING (workflowID)
                         LEFT JOIN step_dependencies USING (stepID)
                         WHERE recordID=:recordID
-                        AND count > 0
-                        AND workflowID > 0';
+                            AND count > 0
+                            AND workflowID != 0
```

Comment:
Is there a special use case that I am not seeing for workflow id 0?

Reply:
workflowID == 0 means a workflow is not associated with the form.

Looking at this more broadly, it might be safe to remove it but I didn't want to expand the scope of this PR. In a separate story, it might be better to check for workflowID != 0 earlier in the function and prevent submission if the workflow was decoupled from the form.